crowbar_framework/config/routes.rb

```diff
@@ -72,6 +72,10 @@
   get "utils/import/:id", controller: "support", action: "import", constraints: { id: /[^\/]+/ }, as: "utils_import"
   get "utils/upload/:id", controller: "support", action: "upload", constraints: { id: /[^\/]+/ }, as: "utils_upload"
   get "utils/restart/:id(.:format)", controller: "support", action: "restart", as: "restart"
+  get "utils/repositories", controller: "repositories", action: "index", as: "repositories"
+  post "utils/repositories", controller: "repositories", action: "sync", as: "repo_sync"
```

Comment:
To keep it like the othger routes you should write `as: "sync_repositories"`, but anyway this route is not really RESTful. IMHO it should be just `get "utils/repositories/sync", controller: "repositories", action: "sync", as: "sync_repositories"`, maybe a POST instead of a GET in that case.